tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -3805,86 +3760,7 @@ GetCudnnFusedOperationGraph(
       new cudnn_frontend::OperationGraph(std::move(op_graph)));
 }
 
-port::StatusOr<std::unique_ptr<cudnn_frontend::ExecutionPlan>>
```

Comment:
Can this be removed in a separate PR to keep history tidy (or just reverted and I can remove it internally)?